psst-gui/src/webapi/client.rs

```diff
@@ -452,75 +808,247 @@ impl WebApi {
+            .query("operationName", "homeSection")
+            .query("variables", &json_query.0.to_string())
+            .query("extensions", &json_query.1.to_string());
+
+        // Extract the playlists
+        let result = self.load_and_return_home_section(request)?;
+        
+        Ok(result)
+    }
+
+    // Shows
+    pub fn your_shows(&self) -> Result<MixedView, Error> {
+        // 0JQ5DAnM3wGh0gz1MXnu3N -> Your shows
+        let json_query = self.build_home_request("spotify:section:0JQ5DAnM3wGh0gz1MXnu3N");
+        let request = self.get("pathfinder/v1/query", Some("api-partner.spotify.com"))?
+            .query("operationName", "homeSection")
+            .query("variables", &json_query.0.to_string())
+            .query("extensions", &json_query.1.to_string());
+
+        let result = self.load_and_return_home_section(request)?;
+        
+        Ok(result)
+    }
+
+    pub fn shows_that_you_might_like(&self) -> Result<MixedView, Error> {
+        // 0JQ5DAnM3wGh0gz1MXnu3P -> Shows that you might like
+        let json_query = self.build_home_request("spotify:section:0JQ5DAnM3wGh0gz1MXnu3P");
+        let request = self.get("pathfinder/v1/query", Some("api-partner.spotify.com"))?
+            .query("operationName", "homeSection")
+            .query("variables", &json_query.0.to_string())
+            .query("extensions", &json_query.1.to_string());
+
+        let result = self.load_and_return_home_section(request)?;
+        
+        Ok(result)
+    }
+
+    /* EPISODES, IMPLEMENT THIS TO REDO THE PODCAST SECTION
+    // Episodes for you, this needs a different thing, mixedview could include this
+    pub fn new_episodes(&self) -> Result<MixedView, Error> {
+        // 0JQ5DAnM3wGh0gz1MXnu3K -> New episodes
+        let json_query = self.build_home_request("spotify:section:0JQ5DAnM3wGh0gz1MXnu3K");
+        let request = self.get("pathfinder/v1/query", Some("api-partner.spotify.com"))?
+            .query("operationName", "homeSection")
+            .query("variables", &json_query.0.to_string())
+            .query("extensions", &json_query.1.to_string());
+
+        // Extract the playlists
+        let result = self.load_and_return_home_section(request)?;
+        
+        Ok(result)
+    }
+
+    // Episodes for you, this needs a different thing, mixedview could include this
+    // 
+    pub fn episode_for_you(&self) -> Result<MixedView, Error> {
+        // 0JQ5DAnM3wGh0gz1MXnu9e -> Episodes for you
+        let json_query = self.build_home_request("spotify:section:0JQ5DAnM3wGh0gz1MXnu9e");
+        let request = self.get("pathfinder/v1/query", Some("api-partner.spotify.com"))?
+            .query("operationName", "homeSection")
+            .query("variables", &json_query.0.to_string())
+            .query("extensions", &json_query.1.to_string());
+
+        // Extract the playlists
+        let result = self.load_and_return_home_section(request)?;
+        Ok(result)
+    }
+    */
```

Comment:
This is okay to keep in, but maybe the commenting style can be made a little more consistent :)